x/exchange/fulfillment.go

```diff
@@ -881,3 +881,58 @@ func getPriceTransfer(f *orderFulfillment) (*Transfer, error) {
 	// This is here in case a new SubTypeI is made that isn't accounted for in here.
 	panic(fmt.Errorf("%s order %d: unknown order type", f.GetOrderType(), f.GetOrderID()))
 }
+
+// NetAssetValue is a total of assets and the price they sold for.
+type NetAssetValue struct {
```

Comment:
I can see this easily getting mixed up with the markertypes.NetAssetValue especially since it uses the MarkerKeeper.

Reply:
I'm open to suggestions. I'm having trouble thinking of an accurate other name for it.

My original thinking was to call it `NetAssetValue` since it's intimately related to the marker's version. I couldn't just use the marker module's version because it doesn't have the asset denom in it, though. The only time the marker's version is used in the exchange module is the one function that makes the calls to `AddSetNetAssetValues`.

Reply:
Works for me. It's only in one spot and we can rename it at a later time if needed.